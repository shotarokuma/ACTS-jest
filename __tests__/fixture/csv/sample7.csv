Column1;Column2;Column3;Column4;Column5;Column6;Column7;Column8;Column9;Column10;Column11;Column12;Column13;Column14;Column15;Column16;Column17;Column18;Column19;Column20
;;;;10;false;;;;;;;;false;19;;;55;false;false
11;;;;;false;;;;false;56;;29;;true;63;;;;42
true;22;55;80;10;false;23;;56;60;;;76;;54;;;false;8;false
;;;;61;91;19;27;;false;;;;false;;false;57;7;78;
3;;false;;64;;;;;;9;false;;26;false;;;;;
;false;62;true;true;false;78;;;false;23;;false;;63;84;;;true;false
;false;0;;84;false;;;false;47;;75;;;;false;;true;;
;50;;true;true;false;;false;;61;88;;11;;false;;;56;;29
true;;;;;;32;false;;;;false;false;;;true;true;true;;
false;86;12;;;;;;;;;;;true;false;;;;;
32;;59;93;;;true;69;false;;;;false;;true;true;31;;;false
true;;;;;;;;74;true;35;false;72;;;;true;;;
;21;true;false;70;;;;1;8;true;40;true;63;2;;41;17;44;false
22;;;;;;;;true;66;false;;;70;;;;24;;
1;;;96;;true;;true;false;false;58;;false;58;53;;;;false;false
2;39;;;0;;true;true;;false;47;4;57;;;;81;false;;false
;;;;;94;;;;;34;false;;;;;true;;false;true
75;false;;;;;;;true;;62;false;;false;false;false;false;;true;
false;;false;;96;26;15;19;34;3;43;;81;;;25;12;;51;false
true;;89;;false;;;;45;;false;;;;;;37;;;
;;;94;17;70;96;69;;;48;;89;true;false;;;7;;
;7;80;7;53;65;;52;;86;51;;62;;;;;35;61;true
45;;false;true;;;;false;;;;;;93;51;;59;;true;
;;;49;;true;;;;;true;;;63;39;9;;;;88
;24;true;;true;;;;31;true;26;83;;54;;19;false;;;81
;;;;false;;82;42;17;;false;false;;;false;;false;90;;false
;4;;;44;;;false;false;true;false;98;;;10;;;true;;false
62;43;;false;;;67;68;true;3;;26;;true;true;true;;89;15;
;78;28;false;false;;49;0;;;;;;;88;28;11;8;;true
;;;;;;;13;6;false;;;;;true;;;true;false;
