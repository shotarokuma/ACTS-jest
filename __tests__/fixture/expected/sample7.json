[
{"Column1":"","Column2":"","Column3":"","Column4":"","Column5":10,"Column6":false,"Column7":"","Column8":"","Column9":"","Column10":"","Column11":"","Column12":"","Column13":"","Column14":false,"Column15":19,"Column16":"","Column17":"","Column18":55,"Column19":false,"Column20":false},
{"Column1":11,"Column2":"","Column3":"","Column4":"","Column5":"","Column6":false,"Column7":"","Column8":"","Column9":"","Column10":false,"Column11":56,"Column12":"","Column13":29,"Column14":"","Column15":true,"Column16":63,"Column17":"","Column18":"","Column19":"","Column20":42},
{"Column1":true,"Column2":22,"Column3":55,"Column4":80,"Column5":10,"Column6":false,"Column7":23,"Column8":"","Column9":56,"Column10":60,"Column11":"","Column12":"","Column13":76,"Column14":"","Column15":54,"Column16":"","Column17":"","Column18":false,"Column19":8,"Column20":false},
{"Column1":"","Column2":"","Column3":"","Column4":"","Column5":61,"Column6":91,"Column7":19,"Column8":27,"Column9":"","Column10":false,"Column11":"","Column12":"","Column13":"","Column14":false,"Column15":"","Column16":false,"Column17":57,"Column18":7,"Column19":78,"Column20":""},
{"Column1":3,"Column2":"","Column3":false,"Column4":"","Column5":64,"Column6":"","Column7":"","Column8":"","Column9":"","Column10":"","Column11":9,"Column12":false,"Column13":"","Column14":26,"Column15":false,"Column16":"","Column17":"","Column18":"","Column19":"","Column20":""},
{"Column1":"","Column2":false,"Column3":62,"Column4":true,"Column5":true,"Column6":false,"Column7":78,"Column8":"","Column9":"","Column10":false,"Column11":23,"Column12":"","Column13":false,"Column14":"","Column15":63,"Column16":84,"Column17":"","Column18":"","Column19":true,"Column20":false},
{"Column1":"","Column2":false,"Column3":0,"Column4":"","Column5":84,"Column6":false,"Column7":"","Column8":"","Column9":false,"Column10":47,"Column11":"","Column12":75,"Column13":"","Column14":"","Column15":"","Column16":false,"Column17":"","Column18":true,"Column19":"","Column20":""},
{"Column1":"","Column2":50,"Column3":"","Column4":true,"Column5":true,"Column6":false,"Column7":"","Column8":false,"Column9":"","Column10":61,"Column11":88,"Column12":"","Column13":11,"Column14":"","Column15":false,"Column16":"","Column17":"","Column18":56,"Column19":"","Column20":29},
{"Column1":true,"Column2":"","Column3":"","Column4":"","Column5":"","Column6":"","Column7":32,"Column8":false,"Column9":"","Column10":"","Column11":"","Column12":false,"Column13":false,"Column14":"","Column15":"","Column16":true,"Column17":true,"Column18":true,"Column19":"","Column20":""},
{"Column1":false,"Column2":86,"Column3":12,"Column4":"","Column5":"","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"","Column11":"","Column12":"","Column13":"","Column14":true,"Column15":false,"Column16":"","Column17":"","Column18":"","Column19":"","Column20":""},
{"Column1":32,"Column2":"","Column3":59,"Column4":93,"Column5":"","Column6":"","Column7":true,"Column8":69,"Column9":false,"Column10":"","Column11":"","Column12":"","Column13":false,"Column14":"","Column15":true,"Column16":true,"Column17":31,"Column18":"","Column19":"","Column20":false},
{"Column1":true,"Column2":"","Column3":"","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"","Column9":74,"Column10":true,"Column11":35,"Column12":false,"Column13":72,"Column14":"","Column15":"","Column16":"","Column17":true,"Column18":"","Column19":"","Column20":""},
{"Column1":"","Column2":21,"Column3":true,"Column4":false,"Column5":70,"Column6":"","Column7":"","Column8":"","Column9":1,"Column10":8,"Column11":true,"Column12":40,"Column13":true,"Column14":63,"Column15":2,"Column16":"","Column17":41,"Column18":17,"Column19":44,"Column20":false},
{"Column1":22,"Column2":"","Column3":"","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"","Column9":true,"Column10":66,"Column11":false,"Column12":"","Column13":"","Column14":70,"Column15":"","Column16":"","Column17":"","Column18":24,"Column19":"","Column20":""},
{"Column1":1,"Column2":"","Column3":"","Column4":96,"Column5":"","Column6":true,"Column7":"","Column8":true,"Column9":false,"Column10":false,"Column11":58,"Column12":"","Column13":false,"Column14":58,"Column15":53,"Column16":"","Column17":"","Column18":"","Column19":false,"Column20":false},
{"Column1":2,"Column2":39,"Column3":"","Column4":"","Column5":0,"Column6":"","Column7":true,"Column8":true,"Column9":"","Column10":false,"Column11":47,"Column12":4,"Column13":57,"Column14":"","Column15":"","Column16":"","Column17":81,"Column18":false,"Column19":"","Column20":false},
{"Column1":"","Column2":"","Column3":"","Column4":"","Column5":"","Column6":94,"Column7":"","Column8":"","Column9":"","Column10":"","Column11":34,"Column12":false,"Column13":"","Column14":"","Column15":"","Column16":"","Column17":true,"Column18":"","Column19":false,"Column20":true},
{"Column1":75,"Column2":false,"Column3":"","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"","Column9":true,"Column10":"","Column11":62,"Column12":false,"Column13":"","Column14":false,"Column15":false,"Column16":false,"Column17":false,"Column18":"","Column19":true,"Column20":""},
{"Column1":false,"Column2":"","Column3":false,"Column4":"","Column5":96,"Column6":26,"Column7":15,"Column8":19,"Column9":34,"Column10":3,"Column11":43,"Column12":"","Column13":81,"Column14":"","Column15":"","Column16":25,"Column17":12,"Column18":"","Column19":51,"Column20":false},
{"Column1":true,"Column2":"","Column3":89,"Column4":"","Column5":false,"Column6":"","Column7":"","Column8":"","Column9":45,"Column10":"","Column11":false,"Column12":"","Column13":"","Column14":"","Column15":"","Column16":"","Column17":37,"Column18":"","Column19":"","Column20":""},
{"Column1":"","Column2":"","Column3":"","Column4":94,"Column5":17,"Column6":70,"Column7":96,"Column8":69,"Column9":"","Column10":"","Column11":48,"Column12":"","Column13":89,"Column14":true,"Column15":false,"Column16":"","Column17":"","Column18":7,"Column19":"","Column20":""},
{"Column1":"","Column2":7,"Column3":80,"Column4":7,"Column5":53,"Column6":65,"Column7":"","Column8":52,"Column9":"","Column10":86,"Column11":51,"Column12":"","Column13":62,"Column14":"","Column15":"","Column16":"","Column17":"","Column18":35,"Column19":61,"Column20":true},
{"Column1":45,"Column2":"","Column3":false,"Column4":true,"Column5":"","Column6":"","Column7":"","Column8":false,"Column9":"","Column10":"","Column11":"","Column12":"","Column13":"","Column14":93,"Column15":51,"Column16":"","Column17":59,"Column18":"","Column19":true,"Column20":""},
{"Column1":"","Column2":"","Column3":"","Column4":49,"Column5":"","Column6":true,"Column7":"","Column8":"","Column9":"","Column10":"","Column11":true,"Column12":"","Column13":"","Column14":63,"Column15":39,"Column16":9,"Column17":"","Column18":"","Column19":"","Column20":88},
{"Column1":"","Column2":24,"Column3":true,"Column4":"","Column5":true,"Column6":"","Column7":"","Column8":"","Column9":31,"Column10":true,"Column11":26,"Column12":83,"Column13":"","Column14":54,"Column15":"","Column16":19,"Column17":false,"Column18":"","Column19":"","Column20":81},
{"Column1":"","Column2":"","Column3":"","Column4":"","Column5":false,"Column6":"","Column7":82,"Column8":42,"Column9":17,"Column10":"","Column11":false,"Column12":false,"Column13":"","Column14":"","Column15":false,"Column16":"","Column17":false,"Column18":90,"Column19":"","Column20":false},
{"Column1":"","Column2":4,"Column3":"","Column4":"","Column5":44,"Column6":"","Column7":"","Column8":false,"Column9":false,"Column10":true,"Column11":false,"Column12":98,"Column13":"","Column14":"","Column15":10,"Column16":"","Column17":"","Column18":true,"Column19":"","Column20":false},
{"Column1":62,"Column2":43,"Column3":"","Column4":false,"Column5":"","Column6":"","Column7":67,"Column8":68,"Column9":true,"Column10":3,"Column11":"","Column12":26,"Column13":"","Column14":true,"Column15":true,"Column16":true,"Column17":"","Column18":89,"Column19":15,"Column20":""},
{"Column1":"","Column2":78,"Column3":28,"Column4":false,"Column5":false,"Column6":"","Column7":49,"Column8":0,"Column9":"","Column10":"","Column11":"","Column12":"","Column13":"","Column14":"","Column15":88,"Column16":28,"Column17":11,"Column18":8,"Column19":"","Column20":true},
{"Column1":"","Column2":"","Column3":"","Column4":"","Column5":"","Column6":"","Column7":"","Column8":13,"Column9":6,"Column10":false,"Column11":"","Column12":"","Column13":"","Column14":"","Column15":true,"Column16":"","Column17":"","Column18":true,"Column19":false,"Column20":""}
]
